File: app/src/main/java/com/OdiousPanda/thefweather/Model/AQI/Time.java
package com.OdiousPanda.thefweather.Model.AQI;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Time {

    @SerializedName("s")
    @Expose
    public String s;
    @SerializedName("tz")
    @Expose
    public String tz;
    @SerializedName("v")
    @Expose
    public Integer v;

}

File: app/src/main/java/com/odiousPanda/rainbowKt/model/dataSource/aqi/Time.kt
package com.odiousPanda.rainbowKt.model.dataSource.aqi


import com.google.gson.annotations.SerializedName

data class Time(
    @SerializedName("s")
    val s: String,
    @SerializedName("tz")
    val tz: String,
    @SerializedName("v")
    val v: Long
)